docs/source/index.rst

```diff
@@ -15,7 +15,11 @@ programming language, since all functions are accessible through a :ref:`rest-ap
 and :ref:`CLI<cli>`. For convenience, the project also includes a :ref:`python-api`, :ref:`R-api`, 
 and :ref:`java_api`.
 
-Get started using the :ref:`quickstart` or by reading about the :ref:`key concepts<concepts>`.
+Get started using the :ref:`quickstart`, reading about the :ref:`key concepts<concepts>`, or
+running `Python <https://docs.databricks.com/applications/mlflow/quick-start-python.html>`_ &
+`R quickstarts <https://docs.databricks.com/applications/mlflow/quick-start-r.html>`_ using hosted
+MLflow on `Databricks Community Edition <https://docs.databricks.com/getting-started/try-databricks.html#community-edition>`_.
```

Comment:
This change makes the instruction to get started too confusing IMO.. Let's keep those simple and focused on the local mode, which lets you use this with your existing code. You can mention CE as a next step after that for people who want to quickly try a hosted server.